.github/workflows/license_check_java-client.yml

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
No need this file any more, https://github.com/apache/incubator-pegasus/blob/master/.github/workflows/check_license_header.yaml will do this work.